### ISSUE TITLE
Action bar in chat

### ISSUE THREAD

**Issue by SlimeDog** [SirBlobman/CombatLogX#160]
**Describe the bug**
[Action Bar] is being reported in chat.

**To Reproduce**
- Update to Spigot 1.16.4
- Attack a mob
- Watch the chat

**Expected behavior**
Use the action bar, as it did previously.

**Screenshots**
If applicable, add screenshots or videos to help explain your problem.
![Screen Shot 2020-11-04 at 6 44 10 AM](https://user-images.githubusercontent.com/17748923/98126509-8617b780-1e6a-11eb-96c0-7283841b8ea9.png)

**Minecraft Version**
 - Do `/clx version` to find out

**CombatLogX Version:**
```
>clx version
[06:54:17] [Server thread/INFO]: Getting version information for CombatLogX...
[06:54:17] [Craft Scheduler Thread - 64/INFO]: 
[06:54:17] [Craft Scheduler Thread - 64/INFO]: Server Version: git-Spigot-37d799b-3eb7236 (MC: 1.16.4)
[06:54:17] [Craft Scheduler Thread - 64/INFO]: Bukkit Version: 1.16.4-R0.1-SNAPSHOT
[06:54:17] [Craft Scheduler Thread - 64/INFO]: Minecraft Version: 1.16.4
[06:54:17] [Craft Scheduler Thread - 64/INFO]: NMS Version: 1_16_R3
[06:54:17] [Craft Scheduler Thread - 64/INFO]: 
[06:54:17] [Craft Scheduler Thread - 64/INFO]: CombatLogX by SirBlobman
[06:54:17] [Craft Scheduler Thread - 64/INFO]: Installed Version: v10.3.1.0
[06:54:17] [Craft Scheduler Thread - 64/INFO]: Latest Version: vUpdate Checker Disabled!
[06:54:17] [Craft Scheduler Thread - 64/INFO]: 
[06:54:17] [Craft Scheduler Thread - 64/INFO]: Getting expansion versions...
[06:54:17] [Craft Scheduler Thread - 64/INFO]: 
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   Citizens Compatibility v15.3
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   PlaceholderAPI Compatibility v15.1
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   Residence Compatibility v15.0
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   SkyBlock Compatibility v15.3
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   WorldGuard Compatibility v15.2
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   Damage Tagger v15.1
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   Logger v15.1
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   Mob Tagger v15.1
[06:54:17] [Craft Scheduler Thread - 64/INFO]:   Notifier v15.6
```

**Additional context**
Add any other information about the problem here.

**Comment by SirBlobman** [comment]
Thanks for the information, please update SirBlobmanCore to 1.0.5 to enable 1.16.4 support
I will leave this open so you can test it.

**Comment by SirBlobman** [comment]
I will update CombatLogX v10 to 1.16.4 soon. Thanks for the information

**Comment by SlimeDog** [comment]
thanks.

**Comment by SirBlobman** [comment]
Should be fixed in 10.3.1.1 🙂

**Comment by SlimeDog** [comment]
It is. Thanks.
